backend/python/app/services/implementations/auth_service.py

```diff
@@ -102,7 +134,8 @@ def is_authorized_by_role(self, access_token, roles):
             user_role = self.user_service.get_user_role_by_auth_id(
                 decoded_id_token["uid"]
             )
-            return user_role in roles
+            firebase_user = firebase_admin.auth.get_user(decoded_id_token["uid"])
+            return firebase_user.email_verified and user_role in roles
```

Comment:
Minor syntax issue: this was incorrectly accessed as `firebase_user["email_verified"]` in `main`, fixed for the release and will make a separate PR to fix in `main`

Reply:
PR for main, please approve as well https://github.com/uwblueprint/starter-code-v2/pull/123